Automatic labels for distinct issue types

weierophinney opened the pull request:
Following the new RFC template, this patch adds labels (and in one case, a title) for the discrete types:

- BC Break: gets the BC Break label, and a title that also calls attention to it.
- Bug: gets the Bug label.
- Feature Request: gets the Enhancement label.

weierophinney committed:
docs: Automatic labels for distinct issue types

- BC Break: gets the BC Break label, and a title that also calls attention to it.
- Bug: gets the Bug label.
- Feature Request: gets the Enhancement label.

Signed-off-by: Matthew Weier O'Phinney <matthew@weierophinney.net>

boesing commented on .github/ISSUE_TEMPLATE/BC_Break.md:
```diff
@@ -1,6 +1,8 @@
 ---
 name: 💥 BC Break
 about: Have you encountered an issue during upgrade? 💣
+title: "[BC BREAK]: [DESCRIPTION]"
```
Copy & paste from Slack:

```
I wonder if we want to use square brackets for the description in #10 as I think it indicates a "placeholder" where the user has to put the real description in.
So does that mean, that he has to remove the square brackets from BC BREAK aswell and replace that text?

And since we initially adding labels anways, I dont think we need the prefix as the label is right next to the description in the issue overview?

so probably we can omit [BC BREAK] at all as the label indicates that after issue created
```

weierophinney committed:
fix: remove title from BC_Break.md template

- It is redundant with the label.
- There may be confusion about whether or not the reporter should replace the text or not.

Signed-off-by: Matthew Weier O'Phinney <matthew@weierophinney.net>
